### ISSUE TITLE
Add VBScript syntax

### ISSUE THREAD

**Issue by philippe-elsass-deltatre** [sharkdp/bat#2075]
**Syntax:**
VBScript

**Guideline Criteria:**
https://packagecontrol.io/packages/VBScript (80k+ downloads, 15k on Mac/Linux)

Although VBScript isn't that popular on Mac/Linux, the syntax is used for a number of basic-inspired languages like BrightScript.

**Comment by philippe-elsass-deltatre** [comment]
Oh no, it looks like it's an old syntax format and doesn't have a `.sublime-syntax`.

**Comment by keith-hall** [comment]
Have you tried using the ASP syntax bundled with bat? https://github.com/sublimehq/Packages/blob/759d6eed9b4beed87e602a23303a121c3a6c2fb3/ASP/ASP.sublime-syntax

**Comment by philippe-elsass-deltatre** [comment]
@keith-hall good call, but it doesn't work - code highlighting is only active inside specific markup.

**Comment by philippe-elsass-deltatre** [comment]
I could convert the old `tmLanguage` VBScript syntax to `.sublime-syntax` with https://github.com/aziz/SublimeSyntaxConvertor and it seems to be working OK. Certainly cumbersome to set up though.

**Comment by keith-hall** [comment]
> @keith-hall good call, but it doesn't work - code highlighting is only active inside specific markup.

seems like it works fine here
![image](https://user-images.githubusercontent.com/11882719/153940517-0e7ce8bb-0caa-492d-ba31-217b67907f60.png)

**Comment by philippe-elsass-deltatre** [comment]
Hmm OK I was confused that `bat --list-languages` would report `HTML (ASP)` and not `ASP`.

That said, while better than nothing, this ASP syntax configuration (right) seem specialised to basic ASP scripting and is quite limited compared to the VBScript one (left):
<img width="790" alt="image" src="https://user-images.githubusercontent.com/82216401/154021958-c3c5cc14-94a2-48d5-ac35-3154083d4807.png">

**Comment by sharkdp** [comment]
Okay. I'd be in favor of adding the specialized VBScript syntax. What do you think @keith-hall?

**Comment by keith-hall** [comment]
Yes, I think it makes sense to have a specialized VBScript syntax, because although similar to ASP, the ASP syntax definition is very strict.
